base_afdemo/data/calendar_event.csv

```diff
@@ -1,285 +0,0 @@
-id;partner_ids/id;partner_ids/name;attendee_ids/id;attendee_ids/display_name;start;duration;meeting_status;location_id;location;name;description
```

Comment:
Why is this file removed?
What does it have to do with contact_links?

Reply:
@MickeHolm this is what happened. The .mo and .po file extension was not in the .gitignore, so i added it, cleared the cache, then push. The base_afdemo/data/calendar_event.csv is inside the .gitignore file, so that is the reason. I did not add it there, you can see for yourself